server/ast/create_database.go

```diff
@@ -24,18 +24,55 @@ import (
 
 // nodeCreateDatabase handles *tree.CreateDatabase nodes.
 func nodeCreateDatabase(node *tree.CreateDatabase) (*vitess.DBDDL, error) {
+	if len(node.Owner) > 0 {
+		return nil, fmt.Errorf("OWNER clause is not yet supported")
+	}
 	if len(node.Template) > 0 {
-		return nil, fmt.Errorf("templates are not yet supported")
+		return nil, fmt.Errorf("TEMPLATE clause is not yet supported")
 	}
 	if len(node.Encoding) > 0 {
-		return nil, fmt.Errorf("encodings are not yet supported")
+		return nil, fmt.Errorf("ENCODING clause is not yet supported")
+	}
+	if len(node.Strategy) > 0 {
+		return nil, fmt.Errorf("STRATEGY clause is not yet supported")
+	}
+	if len(node.Locale) > 0 {
+		return nil, fmt.Errorf("LOCALE clause is not yet supported")
 	}
 	if len(node.Collate) > 0 {
-		return nil, fmt.Errorf("collations are not yet supported")
+		return nil, fmt.Errorf("LC_COLLATE clause is not yet supported")
 	}
 	if len(node.CType) > 0 {
-		return nil, fmt.Errorf("ctypes are not yet supported")
+		return nil, fmt.Errorf("LC_CTYPE clause is not yet supported")
+	}
+	if len(node.IcuLocale) > 0 {
+		return nil, fmt.Errorf("ICU_LOCALE clause is not yet supported")
+	}
+	if len(node.IcuRules) > 0 {
+		return nil, fmt.Errorf("TEMPLATE clause is not yet supported")
+	}
+	if len(node.LocaleProvider) > 0 {
+		return nil, fmt.Errorf("LOCALE_PROVIDER clause is not yet supported")
+	}
+	if len(node.CollationVersion) > 0 {
+		return nil, fmt.Errorf("COLLATION_VERSION clause is not yet supported")
 	}
+	if len(node.Tablespace) > 0 {
+		return nil, fmt.Errorf("TABLESPACE clause is not yet supported")
+	}
+	if node.AllowConnections != nil {
```

Comment:
For these that have default values (from your comment), I'd add a `TODO` that states what the default value should be for when we eventually implement them. It's very possible that the comment may be glanced over. You could also choose to set it in the parser as well (if it's not provided, then we set the expression to a `-1` literal)